ql/src/java/org/apache/hadoop/hive/ql/txn/compactor/Compactor.java

```diff
@@ -59,7 +59,7 @@ static long getCompactorTxnId(Configuration jobConf) {
    * @param dir provides ACID directory layout information
    * @throws IOException compaction cannot be finished.
    */
-  void run(HiveConf hiveConf, Table table, Partition partition, StorageDescriptor storageDescriptor,
+  boolean run(HiveConf hiveConf, Table table, Partition partition, StorageDescriptor storageDescriptor,
```

Comment:
Please describe the meaning of the return value in the javadoc

Reply:
Described. Done.